geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/executor/RedisResponse.java

```diff
@@ -71,6 +71,14 @@ public static RedisResponse string(byte[] byteArray) {
     return new RedisResponse((buffer) -> Coder.getSimpleStringResponse(buffer, byteArray));
   }
 
+  public static RedisResponse floatDouble(byte[] byteArray) {
+    return new RedisResponse((buffer) -> Coder.getSimpleStringResponse(buffer, byteArray));
+  }
+
+  public static RedisResponse floatDouble(String stringValue) {
+    return new RedisResponse((buffer) -> Coder.getSimpleStringResponse(buffer, stringValue));
+  }
+
```

Comment:
I think the semantics of this class and its static methods is intended to mimic the RESP protocol. From that perspective, seeing `RedisResponse.floatDouble` might lead someone to think that RESP can represent a float/double. Prefer instead a `string(double)` method which converts a double into the appropriate RESP response (string in this case).

As an aside, I see that we have `bigDecimal` here which should be changed to adhere to the same semantics (but not in this PR).